web/src/components/Editor/Expression/hooks/useAutoComplete.ts

```diff
@@ -0,0 +1,103 @@
+import {useCallback} from 'react';
+import {uniqBy} from 'lodash';
+import {CompletionContext} from '@codemirror/autocomplete';
+import {syntaxTree} from '@codemirror/language';
+import {completeSourceAfter, parserList, Tokens} from 'constants/Editor.constants';
+import {useAppStore} from 'redux/hooks';
+import AssertionSelectors from 'selectors/Assertion.selectors';
+import SpanSelectors from 'selectors/Span.selectors';
+
+const {demoEndpoints = '{}'} = window.ENV || {};
+const {PokeshopHttp = ''}: Record<string, string> = JSON.parse(demoEndpoints);
```

Comment:
You will need to use the new Env util. `demoEndpoints` is an object in window.ENV (not a string)